feat(guides): Create New Definitions Page (#2059)
TRaSH-Guides/Guides: docs/Glossary/index.md

# Glossary

This glossary provides definitions of commonly used terms and phrases, organized by application or function.

---

## Media Management Applications

**Radarr**, **Sonarr**, and **Lidarr** are applications designed to manage your media libraries:

- **Radarr**: For movies
- **Sonarr**: For TV shows
- **Lidarr**: For music

### Custom Format

> A feature that matches specific aspects of a release based on information extracted from its name. Users choose the matching criteria, and Custom Formats can be assigned scores on a per-[Quality Profile](#quality-profile) basis.

Reviewer: maybe add that the info could also come from metadata of the release? of course only after/on import

Author: > maybe add that the info could also come from metadata of the release? of course only after/on import

Technically it's still the name though, isn't it? The *arr will rename based on the metadata, and the CF matches the name?

Reviewer 2: I think the technical distinction might confuse users, when we could simply note that the CF after import can change based on the file's actual properties rather than simply the name of the torrent on site.

Not all users are familiar with what the metadata of a file actually encompasses, and this could be ambiguous or completely miss users who are familiar with metadata being referred to for things like music which would be manually added versus actual quality/format/codec etc.

No real strong opinion here, just a consideration I wanted to raise. Seems like a more vague description that users familiar with metadata and mediainfo could understand while not convoluting the general public that may not be.

Author: > I think the technical distinction might confuse users, when we could simply note that the CF after import can change based on the file's actual properties rather than simply the name of the torrent on site.
> 
> Not all users are familiar with what the metadata of a file actually encompasses, and this could be ambiguous or completely miss users who are familiar with metadata being referred to for things like music which would be manually added or tags versus actual quality/format/codec etc.
> 
> No real strong opinion here, just a consideration I wanted to raise. Seems like a more vague description that users familiar with metadata and mediainfo could understand while not convoluting the general public that may not be.

I like this approach - if you fancy also having a stab at that during your review!

Author: Covered in zakary's updates

### Quality

Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

### Quality Definitions

> Defines the Minimum, Preferred, and Maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.

Author: No *good* reason! Amended.

### Quality Profile

A defined group of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

---

## Torrents

### Cross-Seed

The act of [seeding](#seeder) the same torrent across multiple [trackers](#tracker). It also refers to a software application of the same name that automates this process.

### Deluge

> A lightweight, free, cross-platform BitTorrent client.

Reviewer: ```suggestion
A lightweight, free, open-source, cross-platform BitTorrent client.
```
should probably also include open-source

### qBit Manage

A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes, such as time and tracker.

> ### qBittorrent

Author: I'll hold this until after zakary's review, then do a recheck for hard alphabetical order which makes more sense for a glossary (as people would be looking up the actual terms).

A free, open-source, cross-platform BitTorrent client.

### Leecher

A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe users who download torrents without sharing them afterward.

### Peer

A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).

### Seeder

BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

### Swarm

The collective group of all BitTorrent users connected to a given torrent, sharing parts of the file. This group includes both [Seeders](#seeder) and [Leechers](#leecher).

### Torrent File

A file containing metadata about the files and folders to be distributed, along with a list of network locations of [trackers](#tracker). Torrent files typically bear the extension `.torrent`.

### Tracker

> Servers that assist participants in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

Reviewer: ```suggestion
Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.
```
I think peers makes more sense here as well

### Perma-Seed

Seed (some of) your torrents permanently. Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places whilst not taking up extra hard disk space.

---

## Usenet

### Atomic Moves

This is a real file move, carried out instantaneously, not the copying of a file from the download folder to the media folder and then deleting the file from the download folder.

### Instant Moves

Same as [Atomic Moves](#atomic-moves).

### DMCA

Also known as: "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.

> When this happens, you often end up with missing/incomplete parts and can't complete the download or repair it with PAR2.

Reviewer: what does this have to do with DMCA?

Reviewer 2: I understand what is being said, but I'm not sure this is relevant to the anchor section as a whole, perhaps this would be better suited in a repair/PAR section to explain the common use-case and failures of these files instead?

Author: > I understand what is being said, but I'm not sure this is relevant to the anchor section as a whole, perhaps this would be better suited in a repair/PAR section to explain the common use-case and failures of these files instead?

When you get to your review, would be good to add 'PAR' as a definition, and perhaps we can expand on it in another section of the guide.

Reviewer 2: Sure.

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

### N&T

> Also known as: "Notice&Takedown". The European equivalent of [DMCA](#dmca).

Reviewer: ```suggestion
Also known as "Notice&Takedown". The European equivalent of [DMCA](#dmca).
```
I think this makes more sense without the colon

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}

### USP

Short for **U**senet **S**ervice **P**rovider

Usenet providers maintain the servers that store Usenet's content and offer you access to download files. [Indexers](#indexers) are required to search for files on Usenet to help you locate specific files within the network.

### Indexers

An indexer is essentially a search engine from which you can download NZB files. These NZB files point to the locations of different parts that make up an entire file, such as a video file. Most indexers require registration, some are invite-only.

When used in the context of Radarr/Sonarr, indexers can refer to both Usenet indexers and [torrent trackers](#tracker).

---

## Shorthand

This section lists equivalent acronyms, initialisms, or shorthand versions for various terms.

### CF

**CloudFlare**, [Custom Formats](#custom-format).

> ### qBit

Author: Added

[qBittorrent](#qbittorrent).

> ### QP

Author: Will wait until after zakary's review before I look at re-ordering things.

# Glossary

This glossary provides definitions of commonly used terms and phrases, organized by application or function.

---

## Media Management Applications

**Radarr**, **Sonarr**, and **Lidarr** are applications designed to manage your media libraries:

- **Radarr**: For movies
- **Sonarr**: For TV shows
- **Lidarr**: For music

### Custom Format

> A feature that matches specific aspects of a release based on information extracted from its name. Users choose the matching criteria, and Custom Formats can be assigned scores on a per-[Quality Profile](#quality-profile) basis.

Reviewer 2: ```suggestion
A feature that uses pattern matching on a release based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
```
Sheesh that was a lot :P

Author: Added a comma.

### Quality

Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

### Quality Definitions

Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.

### Quality Profile

A defined group of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

---

## Torrents

### Cross-Seed

The act of [seeding](#seeder) the same torrent across multiple [trackers](#tracker). It also refers to a software application of the same name that automates this process.

### Deluge

A lightweight, free, open-source, cross-platform BitTorrent client.

### qBit Manage

> A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes, such as time and tracker.

Reviewer 2: ```suggestion
A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes (such as time and tracker.)

Author: Preferred without the brackets, or indeed my previous version with a comma.

### qBittorrent

A free, open-source, cross-platform BitTorrent client.

### Leecher

> A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe users who download torrents without sharing them afterward.

Reviewer 2: ```suggestion
A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.

### Peer

A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).

### Seeder

BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

### Swarm

The collective group of all [Peers](#peer) of a given torrent, sharing parts of the file. This group includes both [Seeders](#seeder) and [Leechers](#leecher).

### Torrent File

A file containing metadata about the files and folders to be distributed, along with a list of network locations of [trackers](#tracker). Torrent files typically bear the extension `.torrent`.

### Tracker

Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

### Perma-Seed

> Seed (some of) your torrents permanently. Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places whilst not taking up extra hard disk space.

Reviewer 2: ```suggestion
Seeding your torrents permanently ("forever".) Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.

---

## Usenet

### Atomic Moves

> This is a real file move, carried out instantaneously, not the copying of a file from the download folder to the media folder and then deleting the file from the download folder.

Reviewer 2: ```suggestion
This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.

### Instant Moves

Same as [Atomic Moves](#atomic-moves).

> 

Reviewer 2: ```suggestion

### PAR (PAR2) Files

PAR and PAR2 files are "Parity Archive Volume" files primarily used in Usenet as a means to attempt repairing incomplete or corrupted files. The presence of these files, however, is not always a guarantee that "reconstruction" of the original data is possible.

```

### DMCA

Also known as: "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.

> When this happens, you often end up with missing/incomplete parts and can't complete the download or repair it with PAR2.

Reviewer 2: ```suggestion
When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

### N&T

Also known as "Notice&Takedown". The European equivalent of [DMCA](#dmca).

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}

### USP

Short for **U**senet **S**ervice **P**rovider

Usenet providers maintain the servers that store Usenet's content and offer you access to download files. [Indexers](#indexers) are required to search for files on Usenet to help you locate specific files within the network.

### Indexers

An indexer is essentially a search engine from which you can download NZB files. These NZB files point to the locations of different parts that make up an entire file, such as a video file. Most indexers require registration, some are invite-only.

When used in the context of Radarr/Sonarr, indexers can refer to both Usenet indexers and [torrent trackers](#tracker).

---

## Shorthand

> This section lists equivalent acronyms, initialisms, or shorthand versions for various terms.

Reviewer 2: ```suggestion
This section lists equivalent acronyms, abbreviations, or shorthand versions for various terms.
```
initialisms? lol

| [**Prowlarr**](../Prowlarr/index.md) | Indexer Management |

### Custom Format

A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.

### Quality

Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

### Quality Definitions

Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.

### Quality Profile

A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

---

## Torrents

### Cross-Seed

> The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process.

Reviewer 3: ```suggestion
The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process. `Cross-Seed` and `crossseed` or `xseed` often refer to the application.  While `cross seed` or `xseed` can refer to the action. Context dependent.
```

Reviewer 2: technically context-dependent is hyphenated according to Grammarly :P

Author: Not sure we need 'context dependent', so removed, and make one more tweak. Good to get the extra terms in there though.

### Deluge

> A lightweight, free, open-source, cross-platform BitTorrent client.

Reviewer 3: ```suggestion
[Deluge](https://deluge-torrent.org/) is a lightweight, free, open-source, cross-platform BitTorrent client.
```

Reviewer 2: lgtm

### qBit Manage

> A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.

Reviewer 3: ```suggestion
[qBit Manage](https://github.com/StuffAnThings/qbit_manage) (aka `qbm` or `qbitmanage` / `qb_manage`)  program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.
```

### qBittorrent

> A free, open-source, cross-platform BitTorrent client.

Reviewer 3: ```suggestion
[qBittorrent ](https://www.qbittorrent.org/) (aka `qb`, `qbit`, `qbittorrent`) is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions due to frequent bugs.
```

### Leecher

A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.

### Peer

A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).

### Seeder

> BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

Reviewer 3: add a note RE open port?

Reviewer 2: Something like

"To actually seed content effectively, it is necessary to have some form of port-forwarding enabled. This can vary depending on whether you use a VPN or not and/or what router or firewall you have. Please consult your router manual or the documentation for your VPN or VPN container for specifics."

Link to port forwarding guide? Although I think the above is more relevant to the specific cases than the guide...

Author: > Something like
> 
> "To seed content effectively, it is necessary to have some form of port-forwarding enabled. This can vary depending on whether you use a VPN or not and/or what router or firewall you have. Please consult your router manual or the documentation for your VPN or VPN container for specifics."
> 
> Link to port forwarding guide? Although I think the above is more relevant to the specific cases than the guide...

I think this would be better added in some form to the port forwarding guide. When I do my pass I'll see if I can reference with a truncated version.

Reviewer 4: In order to torrent, one of the parties must have an open incoming port. If that's not YOU, then it must be the person you're peering from. That is a small % of people, because most people are like you and don't do it.
so basically, you're getting 5-10% of possible seeders on every torrent instead of 100% of seeders on every torrent.

This is something Pearson once mentioned to explain to someone

> [qBittorrent ](https://www.qbittorrent.org/) (aka `qb`, `qbit`, `qbittorrent`) is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.

Reviewer 2: ```suggestion
[qBittorrent](https://www.qbittorrent.org/) (aka `qb`, `qbit`, `qbittorrent`) is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.
```
lint issue.

Author: Resolved in latest push.

### Swarm

The collective group of all [Peers](#peer) of a given torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.

### Torrent File

A file containing metadata and integrity information about the files and folders associated with the torrent, along with a list of the network locations for its [trackers](#tracker). Torrent files typically bear the extension `.torrent`.

### Tracker

Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

### Perma-Seed

> Seeding your torrents permanently ("forever".) Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.

Reviewer: ```suggestion
Seeding your torrents permanently ("forever"). Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.
```

Reviewer 2: oops you're right, that was my mistake :P

---

## Usenet

### Atomic Moves

This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.

### Instant Moves

Same as [Atomic Moves](#atomic-moves).

### PAR (PAR2) Files

PAR and PAR2 files are "Parity Archive Volume" files primarily used in Usenet as a means to attempt repairing incomplete or corrupted files. The presence of these files, however, is not always a guarantee that "reconstruction" of the original data is possible.

### DMCA

> Also known as: "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.

Reviewer: ```suggestion
Also known as "DMCA takedown". The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.
```
missed this initially 😅

Reviewer 2: hey there were a lot... cant be perfect

Reviewer: I was talking about myself lol

> When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.

Reviewer: this still doesn't make sense to me here 🤔

Reviewer 2: Which part specifically?

Reviewer: `When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.`
why would a DMCA cause you to only get your content partially?

Reviewer 2: They sometimes don't or can't identify the entire release across the many parts, and instead just report a few articles.

Some are more easily identifiable in the case of public posts than private indexers nzbs/uploads, but generally speaking my understanding (I don't really deal with this and don't use public indexers) its not the entire release's articles that are taken down.

Reviewer: Ok so this is only really relevant for Usenet, correct?
If that's the case I think it would be a good idea to add that

Reviewer 2: good point

> When this happens, you often end up with missing/incomplete parts and extraction can subsequently fail.

Reviewer 2: ```suggestion
When a DMCA takedown request is issued to a [USP (Usenet Service Provider)](#usp), you may end up with missing or incomplete parts, and extraction can subsequently fail.

Reviewer 3: lgtm

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

### N&T

Also known as "Notice&Takedown". The European equivalent of [DMCA](#dmca).

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}

### USP

Short for **U**senet **S**ervice **P**rovider

Usenet providers maintain the servers that store Usenet's content and offer you access to download files. [Indexers](#indexers) are required to search for files on Usenet and can help you locate specific files within the network.

### Indexers

An indexer is a search engine from which you can download NZB files. NZB files point to the locations of all the different parts that make up an entire file or release. Most indexers require registration and some are invite-only.

When used in the context of Radarr/Sonarr, indexers can refer to both Usenet indexers and [torrent trackers](#tracker).

---

> ## Shorthand

Reviewer 3: I suggest removing this section and adding it within the definitions

Reviewer 2: I agree, I typically don't change anchors when I do grammar, but I think if we were going to have this be its own section at all, something more like "Acroynms and Alternate Identifiers" or something. I dunno, shorthand seems a little weird to use in this context, but I have no real strong opinion on it. Just aesthetics.

Author: Happy to remove this and make them part of each definition. Will do that in a subsequent pass.

---

## A

### Atomic Move

This is a real file move, carried out instantaneously, rather than by copying a file from one location to another and then deleting the file from the original folder.

---

## B

---

## C

### CF

Depending on its context, whether hosting or media management applications, this can refer to **CloudFlare** or [Custom Formats](#custom-format).

### Cross-Seed

The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org){:target="\_blank" rel="noopener noreferrer"} that automates this process.

`Cross-Seed` and `crossseed` or `xseed` often refer to the application, while `cross seed` or `xseed` can refer to the action.

### Custom Format

!!!info

    Also known as: CF

A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile).

> After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.

Author: Doesn't this depend on the naming scheme the user is using? If they're not using the relevant tokens in the naming then those things won't be retained?

> After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.

Reviewer 2: ```suggestion
After the release has been imported, when using the TRaSH recommended naming scheme, the Custom Format score will automatically be updated to match the properties of the _actual_ media file, rather than strictly by its original naming alone.

> After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.

Author: Beautiful.

---

## D

### Deluge

[Deluge](https://deluge-torrent.org/){:target="\_blank" rel="noopener noreferrer"} is a lightweight, free, open-source, cross-platform BitTorrent client.

### DMCA Takedown

!!!info

    Also known as: DMCA

> 

Author: Added link open instructions and amended to wiki link.

The DMCA Takedown process allows creators to send notices to service providers that have material that infringes their copyright.

When a DMCA takedown request is issued to a [USP (Usenet Service Provider)](#usp), you may end up with missing or incomplete parts, and extraction can subsequently fail. DMCA stands for `Digital Millennium Copyright Act`.

[More info from Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

---

## E

---

## F

---

## G

---

## H

---

## I

### Indexer

An indexer is a search engine from which you can download NZB files. NZB files point to the locations of all the different parts that make up an entire file or release. Most indexers require registration and some are invite-only.

When used in the context of Radarr/Sonarr, indexers can refer to both Usenet indexers and [torrent trackers](#tracker).

> ### Instant Move

Author: Think I prefer the singular noun here.

> ### Instant Move

Reviewer 2: Atomic Move**s** - I just tried to keep it consistent.

See [Atomic Moves](#atomic-moves).

---

## J

---

## K

---

## L

### Leecher

A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.

### [Lidarr](../Lidarr/index.md)

> An application designed to help with automation and the management of your music library. One of a group of applications, including Prowlarr, Radarr, and Sonarr, collectively known as the "arrs".

Reviewer 2: ```suggestion
An application designed to help with the automation and management of your music library. One of a group of applications, including Prowlarr, Radarr, and Sonarr, collectively known as the "arrs".

---

## M

---

## N

### Notice & Takedown

!!!info

    Also known as N&T

The European equivalent of [DMCA](#dmca).

[More info from Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}

---

## O

---

## P

### PAR (PAR2) Files

> PAR and PAR2 files are "Parity Archive Volume" files primarily used in Usenet as a means to attempt repairing incomplete or corrupted files. The presence of these files, however, is not always a guarantee that "reconstruction" of the original data is possible.

Reviewer 2: ```suggestion
PAR and PAR2 files, or 'Parity Archive Volume' files, are primarily used in Usenet to help repair incomplete or corrupted files. However, their presence does not always guarantee successful reconstruction of the original data.

### Peer

> A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).

Reviewer 2: ```suggestion
A general term that collectively refers to both [Seeders](#seeder) and [Leechers](#leecher).

### Perma-Seed

> Seeding your torrents permanently ("forever"). Most often, this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.

Reviewer 2: ```suggestion
Seeding your torrents permanently ("forever"). Most often this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.

### [Prowlarr](../Prowlarr/index.md)

> An application designed to help with automation and the management of your [trackers](#tracker) and [indexers](#indexer). One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

Reviewer 2: ```suggestion
An application designed to help with the automation and management of your [trackers](#tracker) and [indexers](#indexer). One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

---

## Q

### qBit Manage

> [qBit Manage](https://github.com/StuffAnThings/qbit_manage){:target="\_blank" rel="noopener noreferrer"} (also known as `qbm` or `qbitmanage` / `qb_manage`) is a program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.

Reviewer 2: ```suggestion
[qBit Manage](https://github.com/StuffAnThings/qbit_manage){:target="\_blank" rel="noopener noreferrer"} (also known as `qbm` or `qbitmanage` / `qb_manage`) is a tool for managing your [qBittorrent](#qbittorrent) instance. It automates actions on torrents meeting specified criteria, such as time seeded and tracker.

### qBittorrent

!!!info

    Also known as: qb, qbit, qbittorrent

[qBittorrent](https://www.qbittorrent.org/){:target="\_blank" rel="noopener noreferrer"} is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.

### Quality

> Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

Reviewer 2: ```suggestion
Refers to the specific quality of a release, describing both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

### Quality Definitions

Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.

### Quality Profile

!!!info

    Also known as: QP

> A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

Reviewer 2: ```suggestion
A defined group, and preferential ordering, of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

---

## R

### [Radarr](../Radarr/index.md)

> An application designed to help with automation and the management of your movie library. One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

Reviewer 2: ```suggestion
An application designed to help with the automation and management of your movie library. One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

---

## S

### Seeder

BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

### [Sonarr](../Sonarr/index.md)

> An application designed to help with automation and the management of your TV Show/Series library. One of a group of applications, including Lidarr, Prowlarr, and Radarr, collectively known as the "arrs".

Reviewer 2: ```suggestion
An application designed to help with the automation and management of your TV Show/Series library. One of a group of applications, including Lidarr, Prowlarr, and Radarr, collectively known as the "arrs".

### Swarm

> The collective group of all [Peers](#peer) of a given torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.

Reviewer 2: ```suggestion
The group of [Peers](#peer), collectively, on a specific torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.

This can also refer to the surge of users downloading a torrent shortly after its release.

---

## T

### Torrent File

> A file containing metadata and integrity information about the files and folders associated with the torrent, along with a list of the network locations for its [trackers](#tracker). Torrent files typically bear the extension `.torrent`.

Reviewer 2: ```suggestion
A file that contains metadata and integrity information about the files and folders associated with a torrent, as well as a list of network locations for its [trackers](#tracker). Torrent files usually have the extension `.torrent`.

### Tracker

> Servers that assist [Peers](#peer) in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

Reviewer 2: ```suggestion
Servers that help [Peers](#peer) locate each other and form an efficient distribution group for a torrent, known as its [swarm](#swarm). The term 'tracker' is also used commonly as a reference to sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

---

## U

### Usenet Service Provider

!!!info

    Also known as: USP

> Usenet providers maintain the servers that store Usenet's content and offer you access to download files. [Indexers](#indexer) are required to search for files on Usenet and can help you locate specific files within the network.

Reviewer 2: ```suggestion
Usenet providers manage the servers that store content and offer subscription-based access for downloading files. [Indexers](#indexer) are required to search for and locate releases on Usenet.

# Glossary

This glossary provides definitions and explanations of commonly used terms, abbreviations, and phrases.

---

## A

### Atomic Move

!!!info

>     Also known as: [Instant Move](#instant-move)

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

This action moves the file instantly, rather than copying it to a new location and deleting it from the original.

---

## B

---

## C

### CF

Depending on its context, whether discussing hosting or media management applications, this can refer to **CloudFlare** or [Custom Formats](#custom-format).

### Cross-Seed

The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org){:target="\_blank" rel="noopener noreferrer"} that automates this process.

`Cross-Seed` and `crossseed` or `xseed` often refer to the application, while `cross seed` or `xseed` can refer to the action.

### Custom Format

!!!info

>     Also known as: [CF](#cf)

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

A feature that applies pattern matching to a release based on information inferred from the release name on the corresponding indexer during a search. Users can customize patterns in a Custom Format, which can then be assigned individual scores within each [Quality Profile](#quality-profile).

After the release has been imported, when using the TRaSH recommended naming scheme, the Custom Format score will automatically be updated to match the properties of the _actual_ media file, rather than strictly by its original naming alone.

---

## D

### Deluge

[Deluge](https://deluge-torrent.org/){:target="\_blank" rel="noopener noreferrer"} is a lightweight, free, open-source, cross-platform BitTorrent client.

### DMCA Takedown

!!!info

>     Also known as: DMCA

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

[More info from Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

---

## E

---

## F

---

## G

---

## H

---

## I

### Indexer

An indexer is a search engine from which you can download NZB files. NZB files point to the locations of all the different parts that make up an entire file or release on Usenet. Most indexers require registration and some are invite-only.

When used in the context of Radarr/Sonarr/Prowlarr (or other media management software), "indexers" can refer to both Usenet indexers and [torrent trackers](#tracker).

### Instant Move

See [Atomic Move](#atomic-move).

---

## J

---

## K

---

## L

### Leecher

A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.

### [Lidarr](../Lidarr/index.md)

An application designed to help with the automation and management of your music library. One of a group of applications, including Prowlarr, Radarr, and Sonarr, collectively known as the "arrs".

---

## M

---

## N

### Notice & Takedown

!!!info

>     Also known as N&T

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

### Peer

A general term that collectively refers to both [Seeders](#seeder) and [Leechers](#leecher).

### Perma-Seed

Seeding your torrents permanently ("forever"). Most often this is done in conjunction with [hard linked](/Hardlinks/Hardlinks-and-Instant-Moves/) files, which allows files to 'exist' in two places without taking up any additional hard disk space.

### [Prowlarr](../Prowlarr/index.md)

An application designed to help with the automation and management of your [trackers](#tracker) and [indexers](#indexer). One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

---

## Q

### qBit Manage

[qBit Manage](https://github.com/StuffAnThings/qbit_manage){:target="\_blank" rel="noopener noreferrer"} (also known as `qbm` or `qbitmanage` / `qb_manage`) is a tool for managing your [qBittorrent](#qbittorrent) instance. It automates actions on torrents meeting specified criteria, such as time seeded and tracker.

### qBittorrent

!!!info

>     Also known as: qb, qbit, qbittorrent

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

[qBittorrent](https://www.qbittorrent.org/){:target="\_blank" rel="noopener noreferrer"} is a free, open-source, cross-platform BitTorrent client. Please note that it is highly recommended to avoid new major and minor (X and X.X) versions until properly validated by the community, as these often contain significant changes and/or bugs.

### Quality

Refers to the specific quality of a release, describing both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.

### Quality Definitions

Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.

### Quality Profile

!!!info

    Also known as: QP

> A defined group, and preferential ordering, of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.

Reviewer: inconsistencies for `TV shows` and `TV Shows`

Author: I think these are OK. One is referring to TV Shows in the plural, and the other is referring to a singular TV Show Library.

---

## R

### [Radarr](../Radarr/index.md)

An application designed to help with the automation and management of your movie library. One of a group of applications, including Lidarr, Prowlarr, and Sonarr, collectively known as the "arrs".

---

## S

### Seeder

BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

### [Sonarr](../Sonarr/index.md)

> An application designed to help with the automation and management of your TV Show/Series library. One of a group of applications, including Lidarr, Prowlarr, and Radarr, collectively known as the "arrs".

Reviewer: inconsistencies for `TV shows` and `TV Shows`

Author: I think these are OK. One is referring to TV Shows in the plural, and the other is referring to a singular TV Show Library.

### Swarm

The group of [Peers](#peer), collectively, on a specific torrent (this includes both [Seeders](#seeder) and [Leechers](#leecher)) who are actively sharing any portion of the torrent.

This can also refer to the surge of users downloading a torrent shortly after its release.

---

## T

### Torrent File

A file that contains metadata and integrity information about the files and folders associated with a torrent, as well as a list of network locations for its [trackers](#tracker). Torrent files usually have the extension `.torrent`.

### Tracker

Servers that help [Peers](#peer) locate each other and form an efficient distribution group for a torrent, known as its [swarm](#swarm). The term 'tracker' is also used commonly as a reference to sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

---

## U

### Usenet Service Provider

!!!info

>     Also known as: USP

Reviewer: inconsistent use of `:` 🤔

Author: Resolved

### qBittorrent

A free, open-source, cross-platform BitTorrent client.

### Leecher

A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe users who download torrents without sharing them afterward.

### Peer

A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).

### Seeder

BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.

### Swarm

The collective group of all BitTorrent users connected to a given torrent, sharing parts of the file. This group includes both [Seeders](#seeder) and [Leechers](#leecher).

### Torrent File

A file containing metadata about the files and folders to be distributed, along with a list of network locations of [trackers](#tracker). Torrent files typically bear the extension `.torrent`.

### Tracker

Servers that assist participants in locating each other and forming efficient distribution groups known as [swarms](#swarm). The term 'tracker' is also informally used for sites that host [.torrent files](#torrent-file) and operate their own tracker servers.

> 

Reviewer 4: ```suggestion

### Perma-Seed

Seed your torrents endlessly, The normal use case would be Perma seed as long as you have the files in your media library. When using [hard links](/Hardlinks/Hardlinks-and-Instant-Moves/) this won't take up extra hard disk space.

```

---

> 

Reviewer 4: ```suggestion

## Usenet

### Atomic Moves

This is a real move, not a copying of a file from the download folder to the media folder and then deleting the file from the download folder.

### Instant Moves

Same as [Atomic Moves](#atomic-moves).

### DMCA

*aka known as "DMCA takedown"*
The DMCA Takedown Process allows creators to send notices to service providers that have material that infringes their copyright.
When this happens, you often end up with missing/incomplete parts and can't complete the download or repair it with PAR2.

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Digital_Millennium_Copyright_Act){:target="\_blank" rel="noopener noreferrer"}

### N&T

*aka known as "Notice&Takedown"*

Often seen as the European version of [DMCA](#dmca).

[More info from the Wikipedia, the free encyclopedia](https://en.wikipedia.org/wiki/Notice_and_take_down){:target="\_blank" rel="noopener noreferrer"}

### USP

Short for **U**senet **S**ervice **P**rovider

Usenet providers maintain the servers that store Usenet's content and offer you access to download files. Indexers are required to search for files on Usenet to help you locate specific files within the network.

### Indexers

An indexer is essentially a search engine from which you can download NZB files. These NZB files point to the locations of different parts that make up an entire file, such as a video file. Most indexers require registration, some are invite-only.

When used in the context of Radarr/Sonarr, indexers mean Usenet indexers and [torrent trackers](#tracker).

---

```
